sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -492,6 +522,28 @@ abstract class ScrollableWidgetListState<T extends ScrollableWidgetList> extends
     ));
   }
 
+  void _setScrollOffset(double newScrollOffset) {
+    super._setScrollOffset(newScrollOffset);
+    config.scrollbarPainter?.scrollOffset = scrollOffset;
+  }
+
+  bool _scrollUnderway = false;
+
+  void _handleDragUpdate(double delta) {
+    super._handleDragUpdate(delta);
+    if (!_scrollUnderway) {
+      _scrollUnderway = true;
+      config.scrollbarPainter?.scrollStarted();
+    }
+  }
+
+  Future _handleDragEnd(Offset velocity) {
+    _scrollUnderway = false;
+    return super._handleDragEnd(velocity).then((_) {
+      config.scrollbarPainter?.scrollEnded() ?? new Future.value();
+    });
```

Comment:
I've added the plumbing for onScrollStart and onScrollEnd listeners. The scrolllableListPainter (was scrollingListPainter) is now driven by dispatchOnScrollStart() etc.